### ISSUE TITLE
Beta or bunch of scammers 

### ISSUE THREAD

**Issue by oonimooni** [ProtonVPN/protonvpn-cli#87]
now i have lost more than 7 days of no hope to connect to any ProtonVpn (except vpn-Tor) due to this issue #77 and no one fixed it yet. 

Protonvpn-cli tool is by far i would call it beta its not in stable cycle at all. so why Protonvpn acting as if they created the greatest tool while its the shittiest vpn tool for linux i ever used and payed for it? 

if u dont have a solution on how to compensate my loose due to ur shitty tool then u r allowing me to call u scammers. at the moment i have lost more than 2 euro wasted on no service from ur side. 

i also wouldnt say these words if u said that protonvpn still in testing cycle or so. because thats give u the right to get these types of errors and i dont have the right to speak because its in testing phase. but since its payable and considered to be stable so please find me a solution upon this.

**Comment by Rafficer** [comment]
First of all, it's hard to make one super stable version for every linux distro and every possible configuration of *nix. There will always be someone where this tool won't work as expected, just because of the big variety.


> at the moment i have lost more than 2 euro wasted on no service from ur side.

You can also use [this](https://protonvpn.com/support/linux-vpn-setup/) guide to set it up manually. That's one solution. 

> i also wouldnt say these words if u said that protonvpn still in testing cycle or so.

ProtonVPN itself is not in testing cycle, but this tool is still in heavy development, it might actually be not wrong to call it beta and I somehow thought they did but can't find it anymore.

> please find me a solution upon this.

Use above linked guide to set up the server you want manually.

Besides all that, I saw that error on my system as well, but the tool still worked. It's not like I can't connect to the internet and my IP doesn't change. It just displays that message, but I don't see any impact from that...

From reading your comments on that thread now, I think you have another issue and the IPv6 message is just something besides that. The IPv6 message comes when trying to disconnect the VPN, but you can't connect in the first place.

Please check if you can connect via above listed guide, I'm happy to assist you troubleshooting if that doesn't work as well, because then the problem resides in either your configuration or your network.

**Comment by mazen160** [comment]
Hi @oonimooni

Thanks for your message. 

For reference, the message you're seeing is there as a warning, not a fatal error exception. This is to inform you that you might have issues with ipv6 handling, so that you should disable it manually. That's all the error means.

As @Rafficer said, it's very difficult to support every single Linux distro, but we do the possible to achieve this.

In case you don't find protonvpn-cli satisfying, feel free to use the default 
openvpn setup.
Guide: https://protonvpn.com/support/linux-vpn-setup/

If you're not satisfied with ProtonVPN, please issue a refund at https://protonvpn.com/support-form

Thanks,
Mazin

**Comment by Rafficer** [comment]
@mazen160 I think he/she has more issues than the IPv6. As you can see [in his/hers comment](https://github.com/ProtonVPN/protonvpn-cli/issues/77#issuecomment-379579591) it shows `Error connecting to VPN` before the warning shows up. I somehow expect there is something wrong with the network.

**Comment by mazen160** [comment]
@Rafficer Yup, the issue ticket is assigned to another team member. He should be fixing it soon.
